ios/FluentUI/Button/Button.swift

```diff
@@ -96,18 +96,6 @@ public enum ButtonStyle: Int, CaseIterable {
     }
 }
 
-// MARK: - Button Colors
-
-public extension Colors {
-    struct Button {
-        public static var background: UIColor = .clear
-        public static var backgroundFilledDisabled: UIColor = surfaceQuaternary
-        public static var borderDisabled: UIColor = surfaceQuaternary
-        public static var titleDisabled: UIColor = textDisabled
-        public static var titleWithFilledBackground: UIColor = textOnAccent
-    }
-}
-
 // MARK: - Button
```

Comment:
Leaving an open comment to stop the automerge to give @laminesm a chance to look too! I'll resolve this and let automerge do its thing if there are no objections later today.